modules/gcloud/src/test/java/org/testcontainers/containers/BigQueryEmulatorContainerTest.java

```diff
@@ -0,0 +1,52 @@
+package org.testcontainers.containers;
+
+import com.google.cloud.bigquery.BigQuery;
+import com.google.cloud.bigquery.BigQueryOptions;
+import com.google.cloud.bigquery.DatasetInfo;
+import com.google.cloud.bigquery.QueryJobConfiguration;
+import com.google.cloud.bigquery.TableResult;
+import org.junit.Test;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class BigQueryEmulatorContainerTest {
+
+    @Test
+    public void test() throws Exception {
+        try (
+            BigQueryEmulatorContainer container = new BigQueryEmulatorContainer("ghcr.io/goccy/bigquery-emulator:0.4.3")
+        ) {
+            container.start();
+
+            String url = String.format("http://%s", container.getEmulatorHttpEndpoint());
+            BigQueryOptions options = BigQueryOptions
+                .newBuilder()
+                .setProjectId("test")
```

Comment:
We probably need `container.getProjectId()`, as "test" is a magical constant here

Reply:
Yep, I think we need to add this for all our gcloud TC implementations 😬